studio/components/interfaces/Organization/AuditLogs/AuditLogs.tsx

```diff
@@ -184,7 +208,35 @@ const AuditLogs = () => {
             </div>
           )}
 
-          {isError && <AlertError error={error} subject="Failed to retrieve audit logs" />}
+          {isError && error.message.endsWith('') ? (
+            <Alert_Shadcn_
+              variant="default"
+              title="Organization audit logs are not available for free or pro organizations"
+            >
+              <IconAlertTriangle className="h-4 w-4 mt-3" />
+              <div className="flex flex-row pt-3 pb-2">
+                <div className="grow">
+                  <AlertTitle_Shadcn_>
+                    Organization audit logs are not available for free or pro organizations
```

Comment:
```suggestion
                    Organization audit logs are not available for Free or Pro organizations
```

Reply:
We could just say "Free or Pro Plans" right? 

The organization detail is more for internal use.